guides/AE2CableManagement.md

```diff
@@ -0,0 +1,77 @@
+Channels can only be 'created' by a ME Controller. Each side of the controller can support up to 32 channels, with there being no way to increase the amount of channels per side. You can add multiple controllers to a single network, however they need to all be directly connected to one another, and can not exceed the size of a 7x7x7 block. 
+
+Channels are fairly simple. Any device that directly interacts with the storage of the network, whether it be exporting items, importing items, acting as an interface to the network, whatever it may be, will use a channel. This means if you were to use a glass cable to power inscribers, the inscribers will not use channels, as they do not interact with the system besides with power. However, connecting to a 'blank' interface (an interface with nothing in it) will still count as a channel.
+
+Do note that molecular assemblers do not directly take items from the network nor import items items into the network, as it uses the interface as a proxy. Therefore, molecular assemblers do not use a channel.
+
+# Cable Management
+
+Alright, now to the fun stuff: P2Ps and cable management. I'll go over P2Ps first, as they are required for cable management.
+
+## P2P
+
+P2P channels have 6 different types: FE, Light, Redstone, Fluid, Item, and ME. For this guide, we will only use the ME P2P, as those are the only ones that will interact with the ME system itself. To bind a P2P to another, you will first need a memory card and then you need to shift right-click on the input cable, and then right-click on the output P2P.
+
+**DO NOT USE THE ADVANCED MEMORY CARD, THIS IS KNOWN TO CAUSE CRASHES**
+
+P2Ps by themself do not do anything. They can are often times used alongside a subnetwork, however it is possible to route it on a single network. I will personally be showing it as if they do use a sepearate network, as I believe that looks better, and helps for demonstration purposes.
+
+A subnetwork would be any network that is not the main one that works alongside the main one, most likely to perform a specific case. The subnetwork is separate from the main network, meaning autocrafting, security, storage, and controllers are separate from one another.
+
+The base purpose of a P2P channel is meant to be a logical management. This guide isn't going to be using that much, but here is a website with a description of how the P2P was designed, and what purpose it is intended to have: https://appliedenergistics.github.io/features/me-network/channels
+
+## Actual Cable Management
+
+Now, the way I described that P2P is not the way we will be using P2P. The way we will be using P2P is to condense channels to reduce the amount of wiring necessary to run around the base, specifically anywhere from the normal 32 channels on a line, upwards to 1024 channels on a single line. The way this works is a little complicated, so here is an image, in which I'll explain what is going on.
+
+![P2PExample](files/AE2CableManagement/P2PExample.png)
```

Comment:
Should mention that you do not need to have a dense case before the input p2p, that is only for if you want to see how many channels that particular p2p is using.